shell/app/modules/project/pages/test-manage/case/header/import-file.tsx

```diff
@@ -35,7 +35,7 @@ const ImportFile = ({ afterImport }: IProps) => {
 
   const onSuccess = ({ successCount }: any) => {
     if (isNumber(successCount)) {
-      message.success(i18n.t('project:imported {total} item successfully', { total: successCount }));
+      message.success(i18n.t('project:importing'));
```

Comment:
will backend still return count? the message should be more meaningful like: start importing, please view detail in records.

Reply:
The successCount returned by backend will only be 0.